jobs/bits-service/templates/bits_config.yml.erb

```diff
@@ -56,3 +56,12 @@ logging:
 public_endpoint: <%= p("bits-service.public_endpoint") %>
 secret: <%= p("bits-service.secret") %>
 skip_cert_verify: true
+<% if p("bits-service.cc_updates.ca_cert") != "" && p("bits-service.cc_updates.ca_cert") != nil &&
```

Comment:
Could this logic be simplified with the help of [if_p](http://bosh.io/docs/reference/jobs.html) maybe?